runtime/doc/lsp.txt

```diff
@@ -2143,23 +2161,26 @@ start({cmd}, {cmd_args}, {dispatchers}, {extra_spawn_params})
 
     Parameters: ~
       • {cmd}                 (string) Command to start the LSP server.
-      • {cmd_args}            (table) List of additional string arguments to
-                              pass to {cmd}.
-      • {dispatchers}         (table|nil) Dispatchers for LSP message types.
-                              Valid dispatcher names are:
+      • {cmd_args}            (string[]) List of additional string arguments
+                              to pass to {cmd}.
+      • {dispatchers}         vim.lsp.rpc.Dispatchers ? Dispatchers for LSP message types. Valid
```

Comment:
```suggestion
      • {dispatchers}         vim.lsp.rpc.Dispatchers? Dispatchers for LSP message types. Valid
```

Reply:
This is the autogenerated text by `scripts/gen_vimdoc.py`, I don't think I can simply change it like that

Reply:
I see. then please never mind.